packages/grafana-o11y-ds-frontend/src/TemporaryAlert.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/css';
+import React, { useEffect } from 'react';
+
+import { Alert, AlertVariant, useTheme2 } from '@grafana/ui';
+
+export const TemporaryAlert = (props: {
+  setVisible: (visible: boolean) => void;
```

Comment:
I am wondering why do we need this `setVisible`/`visible` prop. I think that it would be easier for developers to use this if this component just internally managed this. If you render it, then it would be visible for ~3s and then it wouldn't render anymore. No need to pass visible/setVisible. Only `text` and `severity`.

What do you think?